### ISSUE TITLE
divid by 0 in interp()

### ISSUE THREAD

**Issue by garyfeng** [jackparmer/colorlover#7]
The following code

```
import colorlover as cl
cl.interp( cl.scales['3']['div']['RdYlBu'], 5) 
```

generates:

```
---------------------------------------------------------------------------
ZeroDivisionError                         Traceback (most recent call last)
<ipython-input-8-14a39fa30bf3> in <module>()
      1 import colorlover as cl
----> 2 cl.interp( cl.scales['3']['div']['RdYlBu'], 5)

/Users/garyfeng/anaconda/lib/python2.7/site-packages/colorlover/__init__.py in interp(scl, r)
   1821         # garyfeng: c_i could be rounded up so scl[c_i+1] will go off range
   1822         #c_i = int(i*math.floor(SCL_FI)/round(r[-1])) # start color index
-> 1823         c_i = int(math.floor(i*math.floor(SCL_FI)/round(r[-1]))) # start color index
   1824         hsl_o = rgb_to_hsl( scl[c_i] ) # convert rgb to hls
   1825         hsl_f = rgb_to_hsl( scl[c_i+1] ) if c_i < len(scl)-1 else hsl_o

ZeroDivisionError: float division by zero

```

Apparently `round(r[-1])==0`.

**Comment by garyfeng** [comment]
Oh, forgot to say I used the code I posted at #6 to fix the out-of-bound error.

**Comment by jonmmease** [comment]
Fixed by #9 and will be released in version 0.3.  Thanks @garyfeng!
